libs/langchain/langchain/schema/runnable/configurable.py

```diff
@@ -239,16 +239,15 @@ class RunnableConfigurableAlternatives(DynamicRunnable[Input, Output]):
 
     @property
     def config_specs(self) -> Sequence[ConfigurableFieldSpec]:
-        alt_keys = self.alternatives.keys()
-        which_keys = tuple(Literal[k] for k in alt_keys) + (  # type: ignore
-            Literal["default"],
+        which_enum = enum.StrEnum(  # type: ignore[misc]
```

Comment:
Our of curiosity, what is the error this is suppressing?

Reply:
Pylance/mypy doesn't like a non-constant value being passed in as the name of the enum, but from checking the code this doesnt raise an issue